aries_cloudagent/messaging/issue_credential/v1_0/manager.py

```diff
@@ -73,6 +63,12 @@ def context(self) -> InjectionContext:
 
         """
 
+        credential_definition_id = credential_proposal.cred_def_id
```

Comment:
The proposal will not include a cred def id in future - rather, an issuer and schema id - so this will need some touching up at that time. Until then, it's fine.